app/generic/verifier/verifier.py

```diff
@@ -0,0 +1,154 @@
+import logging
+from enum import Enum
+from typing import Literal
+
+from aries_cloudcontroller import AcaPyClient
+from fastapi import APIRouter, Depends
+
+from app.dependencies import agent_selector
+from app.generic.verifier.facades.acapy_verifier import Verifier
+from app.generic.verifier.facades.acapy_verifier_v1 import VerifierV1
+from app.generic.verifier.facades.acapy_verifier_v2 import VerifierV2
+from app.generic.verifier.models import (
+    AcceptProofRequest,
+    CreateProofRequest,
+    PresentationExchange,
+    ProofRequestProtocolVersion,
+    RejectProofRequest,
+    SendProofRequest,
+)
+
+logger = logging.getLogger(__name__)
+
+
+router = APIRouter(prefix="/generic/proof", tags=["proof"])
+
+ProtocolVersion = Literal[
+    ProofRequestProtocolVersion.v10.value, ProofRequestProtocolVersion.v20.value
+]
```

Comment:
Is this used? I think we can just use the protocol version values right?

Reply:
hmm. yeah we could also do that.